src/latin_language_model.js

```diff
@@ -194,7 +194,10 @@ export default class LatinLanguageModel extends LanguageModel {
       pronounClassRequired: false
     }
     if (inflection.hasOwnProperty(Feature.types.part)) {
-      if (inflection[Feature.types.part].value === Constants.POFS_PRONOUN) {
+      if (inflection[Feature.types.part].value === Constants.POFS_VERB) {
+        grammar.fullFormBased = true
```

Comment:
I am just wondering if we will run into any problems with having both constraints (fullFormBased and suffixBased) be true. But it if you have tested and both irregular and regular verbs are working as expected, then I guess this is okay.

Reply:
Yes, I have tested - I will place here some screenshots - may be you will detect some errors - I couldn't

Ordinary verb
![image](https://user-images.githubusercontent.com/12377640/42169030-fbca3f3c-7e55-11e8-96bf-13dbcfd835d5.png)

Ordinary and Irregular
![image](https://user-images.githubusercontent.com/12377640/42169083-20b38182-7e56-11e8-9394-ca8bab25a774.png)

![image](https://user-images.githubusercontent.com/12377640/42169121-349b049a-7e56-11e8-9dc0-5b89c8b0b2f7.png)